private/bufpkg/bufplugin/bufpluginref/bufpluginref.go

```diff
@@ -107,7 +119,7 @@ func parsePluginIdentityComponents(path string) (remote string, owner string, pl
 		return "", "", "", newInvalidPluginIdentityStringError(path)
 	}
 	plugin = strings.TrimSpace(slashSplit[2])
-	if plugin == "" {
+	if plugin == "" || strings.ContainsRune(plugin, ':') {
```

Comment:
Do we still need this `:` check?

Reply:
This felt like a good safeguard to prevent someone from attempting to parse a PluginReference as a PluginIdentity. Previously that would work without errors, even though the plugin name should never contain a `:` separator.